packages/radix-vue/src/Tooltip/TooltipProvider.vue

```diff
@@ -37,6 +38,14 @@ export interface TooltipProviderProps {
    * @defaultValue false
    */
   disableClosingTrigger?: boolean
+  /**
+   * Prevent the tooltip from opening if the focus did not come from
+   * the keyboard by using the `:focus-visible` selector.
+   * This is useful if you want to avoid opening it when switching
+   * browser tabs or closing a dialog.
+   * @defaultValue false
+   */
+  keyboardFocusOnly?: boolean
```

Comment:
`keyboardFocusOnly` naming might confuse user with changing the tooltip behavior to only "keyboard focus".
Maybe something like `disableNonKeyboardFocus`?

Reply:
Updated the prop name to `ignoreNonKeyboardFocus`.

Reply:
Lovely!! Thanks @remonke ! 😁